src/Git/PickLastMinorVersionFromCollection.php

```diff
@@ -24,6 +24,10 @@ public function assert(Version $version): bool
                 return ! $version->isPreRelease();
             }
         });
+        
+        if ($stableVersions->isEmpty()) {
+            throw new \Exception('Your library does not have any minor version');
```

Comment:
Is a crash really the best approach? Perhaps it should be "green" when no BC breaks are detected instead?

Reply:
Thanks for the feedback. That's why I kinda put it as a draft with WIP. And yes when there is no stable version - there won't be any BC breaks. Note or Info message could be applied.

Stay tuned 😄